doc/source/whatsnew/v0.25.0.rst

```diff
@@ -96,7 +96,7 @@ Timedelta
 Timezones
 ^^^^^^^^^
 
--
+- Bug in :func:`to_datetime` with ``utc=True` and datetime strings that would apply previously parsed UTC offsets to subsequent arguments (:issue:`24992`)
```

Comment:
small nitpick: missing the second backtick at the end of <code>``utc=True`</code>

```suggestion
- Bug in :func:`to_datetime` with ``utc=True`` and datetime strings that would apply previously parsed UTC offsets to subsequent arguments (:issue:`24992`)
```